app/models/bus_stop.rb

```diff
@@ -6,7 +6,6 @@
 class BusStop < ApplicationRecord
   has_paper_trail
 
-  has_and_belongs_to_many :routes
```

Comment:
But a bus stop does have and belong to many routes, though. Are we saying it's impossible to define this association now that we have a join model?

Reply:
Idk. We have this line https://github.com/umts/stop-project/pull/139/files#diff-a4c987345e1391c4e1f6ff9be1c6ed5aL18

Reply:
Oh, I see, sorry. That's fine then. We certainly can't have both.